pkg/metrics/metrics_test.go

```diff
@@ -8,17 +8,21 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+const (
+	defaultPriority = "0"
+)
+
 func TestCache(t *testing.T) {
 	m := New(logrus.NewEntry(logrus.New()))
 
 	for i, typ := range []string{"init", "container"} {
 		version := fmt.Sprintf("0.1.%d", i)
-		m.AddImage("namespace", "pod", "container", typ, "url", true, version, version)
+		m.AddImage("namespace", "pod", "container", typ, "url", true, version, version, defaultPriority)
 	}
 
```

Comment:
If we do need this, (see comment in `metrics.go`) Can we have a test that covers the setting and reflection of a different priority and validate that the metric set reflects the value given.